src/main/java/uk/gov/hmcts/ccd/domain/model/casedataaccesscontrol/RoleMatchingResult.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol;
+
+import lombok.Data;
+
+@Data
+public class RoleMatchingResult {
+    boolean validDate;
```

Comment:
I don't see this as a validation.
I'd suggest renaming to `dateMatched`, `jurisdictionMatched`....